src/views/Project/Details/Frameworks/FrameworkDetail/EditFrameworkModal/FrameworkUsersTable/AddFromSearch/index.js

```diff
@@ -0,0 +1,236 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import produce from 'immer';
+import { _cs } from '@togglecorp/fujs';
+
+import {
+    RequestClient,
+    requestMethods,
+} from '#request';
+
+import NaiveSearchList from '#components/general/NaiveSearchList';
+import Message from '#rscv/Message';
+import UserAddItem from '#components/general/UserAddItem';
+
+import _ts from '#ts';
+import notify from '#notify';
+
+import styles from './styles.scss';
+
+const propTypes = {
+    className: PropTypes.string,
+    frameworkId: PropTypes.number,
+    searchText: PropTypes.string,
+    onSearchChange: PropTypes.func,
+    listGetRequest: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    userAddRequest: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+    setDefaultRequestParams: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+    className: '',
+    frameworkId: undefined,
+    searchText: '',
+    onSearchChange: () => {},
+    listGetRequest: {},
+    userAddRequest: {},
+};
+
+const emptyList = [];
+
+const SearchEmptyComponent = ({ className }) => (
```

Comment:
Should move this before the `propTypes` and `defaultProps` of `AddFrameworkUserFromSearch`